READMEs/datatokens-flow.md

```diff
@@ -68,14 +81,43 @@ from ocean_lib.web3_internal.wallet import Wallet
 private_key = os.getenv('TEST_PRIVATE_KEY1')
 config = ExampleConfig.get_config()
 ocean = Ocean(config)
+print(f"config.network_url = '{config.network_url}'")
+print(f"config.block_confirmations = {config.block_confirmations.value}")
 
-print("create wallet: begin")
+print("Create wallet: begin")
 wallet = Wallet(ocean.web3, private_key, config.block_confirmations, config.transaction_timeout)
-print(f"create wallet: done. Its address is {wallet.address}")
-
-print("create datatoken: begin.")
-datatoken = ocean.create_data_token("Dataset name", "dtsymbol", from_wallet=wallet)
-print(f"created datatoken: done. Its address is {datatoken.address}")
+print(f"Create wallet: done. Its address is {wallet.address}")
+
+print("Create ERC721 token: begin.")
```

Comment:
Suggested tweak: "Create ERC721 token" --> "Create ERC721 data NFT". 

And for everything below: "ERC721 token" --> "data NFT"